well-sample/src/main/java/com/yarolegovich/wellsample/AntiHero.java

```diff
@@ -0,0 +1,63 @@
+package com.yarolegovich.wellsample;
+
+import com.yarolegovich.wellsql.core.Identifiable;
+import com.yarolegovich.wellsql.core.annotation.Column;
+import com.yarolegovich.wellsql.core.annotation.PrimaryKey;
+import com.yarolegovich.wellsql.core.annotation.RawConstraints;
+import com.yarolegovich.wellsql.core.annotation.Table;
+import com.yarolegovich.wellsql.core.annotation.Unique;
+
+@Table
+@RawConstraints({"FOREIGN KEY(HERO_ID) REFERENCES SuperHero(_id)", "FOREIGN KEY(VILLAIN_ID) REFERENCES Villain(_id)"})
+public class AntiHero implements Identifiable {
```

Comment:
Should we rename that class to `StrangeCouple` :trollface: (and drop the name column).